src/main/java/Deadline.java

```diff
@@ -7,6 +7,14 @@ public Deadline(String description, String by) {
         this.by = by;
     }
 
+    public String getName() {
+        return "D";
+    }
+
+    public String timing() {
```

Comment:
Perhaps it would be better to name the method such as getTiming() ?

Reply:
Thank you! Changed :)